src/reader/reader_utils.cairo

```diff
@@ -93,32 +91,61 @@ fn get_borrowing_fees(
+    );
 
-    let funding_fee_amount_per_size_collateral_type_long = CollateralType {
-        long_token: 0, short_token: 0,
-    };
-    let funding_fee_amount_per_size_collateral_type_short = CollateralType {
-        long_token: 0, short_token: 0,
+    let funding_long_short_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.short_token, true // is_long
+    );
+
+    let funding_short_long_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.long_token, false // is_long
+    );
+
+    let funding_short_short_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.short_token, false // is_long
+    );
+
+    let claimable_long_long_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.long_token, true // is_long
+    );
+
+    let claimable_long_short_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.short_token, true // is_long
+    );
+
+    let claimable_short_long_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.long_token, false // is_long
+    );
+
+    let claimable_short_short_token = market_utils::get_funding_fee_amount_per_size(
+        data_store, market.market_token, market.short_token, false // is_long
+    );
+
+    let long_funding = CollateralType {
+        long_token: funding_long_long_token, short_token: funding_long_short_token
     };
-    let claimable_funding_amount_per_size_type_long = CollateralType {
-        long_token: 0, short_token: 0,
+
+    let short_funding = CollateralType {
+        long_token: funding_short_long_token, short_token: funding_short_short_token
     };
-    let claimable_funding_amount_per_size_type_short = CollateralType {
-        long_token: 0, short_token: 0,
+
+    let long_claimable = CollateralType {
+        long_token: claimable_long_long_token, short_token: claimable_long_short_token
     };
 
-    let funding_fee_amount_per_size = PositionType {
-        long: funding_fee_amount_per_size_collateral_type_long,
-        short: funding_fee_amount_per_size_collateral_type_short,
+    let short_claimable = CollateralType {
+        long_token: claimable_short_long_token, short_token: claimable_short_short_token
     };
 
+    let funding_fee_amount_per_size = PositionType { long: long_funding, short: short_funding };
+
     let claimable_funding_amount_per_size = PositionType {
-        long: claimable_funding_amount_per_size_type_long,
-        short: claimable_funding_amount_per_size_type_short,
+        long: long_claimable, short: short_claimable,
     };
-    BaseFundingValues { funding_fee_amount_per_size, claimable_funding_amount_per_size, }
+
+    BaseFundingValues { funding_fee_amount_per_size, claimable_funding_amount_per_size }
```

Comment:
Maybe it's cleaner to initialize as mut with Default and assign each field like in solidity